### PR TITLE
Update styling

### PR DESCRIPTION
**Pull request by allysonjp715**
Still some more styling to come. Based on most recent figma: https://www.figma.com/file/TqqzDnXF4KTBAGUchpt4mQ/Global.Health-Marketing-Website?node-id=278%3A2

Screen Shot 2020-09-01 at 2 56 40 PM
Screen Shot 2020-09-01 at 3 24 46 PM
Screen Shot 2020-09-01 at 3 32 25 PM
Screen Shot 2020-09-01 at 3 32 09 PM

### CODE REVIEW

File: verification/curator-service/ui/src/components/LinelistTable.tsx [hunk 1]
```diff
@@ -774,6 +778,7 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                         pageSize: this.state.pageSize,
                         pageSizeOptions: [5, 10, 20, 50, 100],
                         maxBodyHeight: 'calc(100vh - 20em)',
+                        // selectionProps: { color: 'primary' },
```
Comment (attwad) [inline verification/curator-service/ui/src/components/LinelistTable.tsx:781]: ah! finally found something to comment on. /satisfied

File: verification/curator-service/ui/src/components/AutomatedSourceForm.tsx [hunk 2]
```diff
@@ -135,7 +135,7 @@ export default function AutomatedSourceForm(props: Props): JSX.Element {
                             </Typography>
                         </div>
                         <Form>
-                            <fieldset className={classes.allFormSections}>
+                            <Paper className={classes.allFormSections}>
```
Comment (attwad) [inline verification/curator-service/ui/src/components/AutomatedSourceForm.tsx:138]: you can remove the fieldset custom css that I added to index.css now I believe

**Review by attwad — APPROVED** [review]
Gorgeous

File: verification/curator-service/ui/src/components/AutomatedSourceForm.tsx (hunk 2 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/AutomatedSourceForm.tsx:138]: Done.

**Pull request merged**